toml.abnf

```diff
@@ -48,17 +49,29 @@ comment = comment-start-symbol *non-eol
 keyval = key keyval-sep val
 
 key = simple-key / dotted-key
+val = string / boolean / array / inline-table / date-time / float / integer
 simple-key = quoted-key / unquoted-key
 
-unquoted-key = 1*( ALPHA / DIGIT / %x2D / %x5F ) ; A-Z / a-z / 0-9 / - / _
+;; Unquoted key
+
+unquoted-key = 1*unquoted-key-char
+unquoted-key-char = ALPHA / DIGIT / %x2D / %x5F         ; a-z A-Z 0-9 - _ 
+unquoted-key-char =/ %xC0-D6 / %xD8-F6 / %xF8-FF        ; non-symbol chars in Latin block
+unquoted-key-char =/ %x100-2FF / %x300-37D / %x37F-1FFF ; this excludes GREEK QUESTION MARK, which is basically a semi-colon 
```

Comment:
There seems to be a trailing space after "semi-colon", which could be trimmed.

Reply:
Done, removed